abcclassroom/example-data/sample_config.yml

```diff
@@ -0,0 +1,56 @@
+
+# The GitHub organization to host your GitHub classroom
+# Create a new organization on GitHub and change this to the name you gave it
+organization: earth-analytics-edu
+
+# (Optional) Short name for organization. If set, this gets used to prepend
+# the repositories instead of the full organization name, just to make
+# names shorter and less unwieldy
+organization-shortname: ea
+
+# Name of your course. Will be used to prefix repositories inside the
+# organization. The repositories will be named:
+# https://github.com/<organization>/<courseName>-template
+# https://github.com/<organization>/<courseName>-<studentName>
+# or, if organization-shortname is set:
+# https://github.com/<organization-shortname>/<courseName>-template
+# https://github.com/<organization-shortname>/<courseName>-<studentName>
+course_name: earth-analytics-bootcamp
+
+# Absolute path to the course roster (list of students downloaded from
+# github classroom)
+roster: /Users/me/awesome-course/roster.csv
```

Comment:
i think this should not be an absolute path. @kcranston might have a newer version of this file in her pr??

Reply:
Yes - I've done some refactoring of paths so that you can use either relative or absolute paths (see my updated sample_config).